### PR TITLE
Add netcdf to cog tutorial notebook

### PR DESCRIPTION
**Pull request by j08lue**
## Description
This notebook shows how to convert a NetCDF file to COG, at the example of CMIP6

## What type of example is this?

Tutorial

## Checklist
_The following checklist ensures that our notebooks are internally consistent ([read more](https://nasa-impact.github.io/veda-docs/contributing/doc-and-notebooks))_

- [x] The first cell contains the rendering information with author and data
- [x] The notebook has a **Run this Notebook** section (with filename in link)
- [x] The notebook has an **Approach** section
- [x] All the packages that are imported in the notebooks are used within the notebook
- [x] Any complex geometries are accessed from remote storage
- [x] Each code cell comes after a markdown cell containing explantory text
- [x] All cells in the notebook book have been run in order with a fresh kernel (cell numbers should be 1, 2, 3, 4, ...)
- [x] If the example type is **Dataset** the filename is `<collection_id>.ipynb`

### CODE REVIEW

File: notebooks/tutorials/netcdf-to-cog-cmip6.ipynb [hunk 1]
```diff
@@ -0,0 +1,1349 @@
+{
```
Comment (jsignell) [inline notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    client = boto3.client("s3")
</pre><p>Is this line necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (jsignell) [inline notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1]: <p>typo: help</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (jsignell) [inline notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1]: <p>What is this cell showing? Can you add text or remove?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

**Comment by jsignell** [comment]
Nice! I just have a few nitpicks and in general I would prefer a bit more explanation, but overall this is great!

**Review by anayeaye — APPROVED** [review]
I added a note about a link we may want to provide to explain cog profile settings

File: notebooks/tutorials/netcdf-to-cog-cmip6.ipynb (hunk 1 above)
Comment (j08lue) [reply notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1]: Well spotted. Removed.

Comment (j08lue) [reply notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1]: Fixed, thanks.

Comment (j08lue) [reply notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1]: Good call - added annotation.

File: notebooks/tutorials/netcdf-to-cog-cmip6.ipynb [hunk 2]
```diff
@@ -0,0 +1,1349 @@
...
...q17JSiSMuRdtJ1yhJkO8ko6u4xwEWfv5ibGhZ9oGTy9dg5++cy31H9CwWZhwz6FOzOaPro8pHP/pR9tprL1asWEGt1mqqvulNb+LHP/7xjO5cDxs3YuMco79838s7lkssgTQkOmAor5L4H+/ct5tyK4l1WP69tmivyrzpl8fz5js+iLGCd/zve2dk+5sDbn3weR2TQ50p3OMrJmOrMq2KjBwntm1vQ7l1xZj7dgqNJ82KWk3lZqIfKuW9b9rhxOjjCc9kaK8eFZWhnZ7tdDTF1NW9j2w76f3H4pUL7vMJ3NJXcKSv0riKTGFWmbbFimRGOcJvWjYIQNmezb3+TGJR0iIECOVVvr50JYyA0CKkbbEYgTf/w5MU4UIqLZioYCq+mlNUdXyMggl8a6rgTMqRHV0FXbX87ZSTWo+hWq/187971pRfqx56WBfomtT84he/4JOf/CRRFHUsnz9/Po89tvqzmB42Lxy7y23jqjXt3igr0xqjeYUVWb0s0QOMZM4SINYzU8P+9r5X8OY7PgjAjft9ASksX9/ny3x9ny/PyPY3BxRVmlA4cbBqEwWPnXRSY8TCBcaOd09U0ZmM2ECnHmaq5Kaj3TTGR6esBNlWC6ogJE732qnzaZ/Maq80tYuG//7Ytix7dBuWPboNIa1oh24waiOfhxYSm6B13UbEJqRhKiQmLO0NNJKmjsj89yg1AbltIzQ+bkQIXHyCMsjQIELvO1MY5Ck3niRCRzKtsmXrCTzx8SRGZgLjqzPWt5ysoi0byt+m/DRVUFRzrKsAtWHZ+04pr//x9Wf3qjTrChtAKLwpomtSY4xBaz1u+aOPPrrB1M49bDgUxGbJPa3sj6KMPpxWaOiIxASM6ApNHZKagKG0QqwDRrJoNVvuDoEc/5l88x0f7E2sAbc8uKsPUDTevK4lDl7duPZkaK/iFG2rVvtJEgo1TjczNk9qskypyZK/i22MvT8wjni0/GEmHuF2hEZgMN4YUBIiCYV71oX7sRIQ4mId7n1kW+5+ePtJX5N2vGbhnxg1lTZyEzBqKqRWubaTXxabkMwomjpECoOxkty4zLREB6RGEeeBc1CWxo12C4uUhbbGeM8Z0TLFizTCkxkbudFvtEvZLvxpVFP4Ue/W9JT2Opm8DtqPhBdj4SZ0lZy8btE1w4MfdrEIC79xPmp0ZlrIPUwBGyCle1NE15/
```
[the remark this answers is not in the document]
Comment (j08lue) [reply notebooks/tutorials/netcdf-to-cog-cmip6.ipynb:1986]: Yes, very useful to keep that - I added the link.

**Comment by j08lue** [comment]
Merged - thanks for the reviews!

**Pull request merged**